pkgs/development/tools/quick-lint-js/default.nix

```diff
@@ -12,11 +10,40 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-OxzemfWYItYb4XWpW2tNUn4yZHUIpS8MJXaP8+3z4YY=";
   };
 
+  quick-lint-js-build-tools = buildPackages.stdenv.mkDerivation {
+    pname = "quick-lint-js-build-tools";
+    inherit version src;
+
+    nativeBuildInputs = [ cmake ninja ];
+    doCheck = false;
+
+    cmakeFlags = [
+      "-DQUICK_LINT_JS_ENABLE_BUILD_TOOLS=ON"
+      # Temporary workaround for https://github.com/NixOS/nixpkgs/pull/108496#issuecomment-1192083379
+      "-DCMAKE_SKIP_BUILD_RPATH=ON"
+    ];
+    ninjaFlags = "quick-lint-js-build-tools";
+
+    installPhase = ''
```

Comment:
missing pre and post runHook's

Reply:
Can you share an example of another package running hooks from installPhase? All the examples I've seen look like this (no extra hooks).

Reply:
@strager I think he meant something like this:
https://github.com/ratsclub/nixpkgs/blob/8d136b1d4ed0e5f7d661acd171695f9df030b3fd/pkgs/applications/misc/xmrig/default.nix#L54